e2e/_suites/fleet/features/permission_output_change.feature

```diff
@@ -0,0 +1,36 @@
+@permission_change
+Feature: Permission output change
+Scenarios for Permission Change
+
+@adding-integration-change-permission
+Scenario Outline: Adding the Linux Integration to an Agent ...
+  Given a "<os>" agent is deployed to Fleet with "tar" installer
+    And the agent is listed in Fleet as "online"
+    And the agent get Default Api Key
+  When the "Linux" integration is "added" in the policy
+    And a Linux data stream exists with some data
+  Then verify that Default Api Key is "different"
```

Comment:
I'd prefer this style
```suggestion
  Then the Default Api Key is "different"
```
OTOH, what does "different" mean? Is it a Fleet concept by itself?